lib/verifier.py

```diff
@@ -167,6 +167,22 @@ def _raise_if_valid_tx(cls, raw_tx: str):
         tx = Transaction(raw_tx)
         try:
             tx.deserialize()
+            # Besides deserializing, we perfom additional checks to reduce the
+            # chance that a legitimate merkle node trips us up.
+            # A legitimate txn of 64 bytes can only have 1 input and 1 output.
+            txin, = tx.inputs()
```

Comment:
Hate to nitpick, but you work with Amaury a lot so you are used to it: Can you use `assert` here to make this code easier to read?

Reply:
But I am in love with the `a, = b` construction :-D

Reply:
TBH it took me like 5 extra seconds to realize what's happening there and I got temporarily angry at you. Is that why?  Hehe :)

Ok, you can keep it...